bHY6yh-d1e515/cod-pol7d7_bHY6yh-d1e515.xml

```diff
@@ -64,129 +64,140 @@
          <lb ed="#K"/>De qualitate dei 3
         </p>
         <p xml:id="bHY6yh-d1e515-d1e119">
-          <lb ed="K" n="53"/> <ref xml:id="bHY6yh-d1e515-Rd1e122">IN exodo. ca. 20.</ref> <quote xml:id="bHY6yh-d1e515-Qd1e121" source="http://scta.info/resource/ex20_5@6-12">Ego
-          sum dominus 
-          <lb ed="#K"/>deus fortis et zelotes.</quote>
+          <lb ed="K" n="53"/> <ref xml:id="bHY6yh-d1e515-Rd1e122">In exodo ca 20</ref> <quote xml:id="bHY6yh-d1e515-Qd1e121" source="http://scta.info/resource/ex20_5@6-12">Ego sum dominus 
+          <lb ed="#K"/>deus fortis et zelotes</quote>
         </p>
         <p xml:id="bHY6yh-d1e515-d1e131">
```

Comment:
Let's meet to go over this. I think there is a quote (from the decretals) in your manuscript that is not in the Venice manuscript and therefore paragraph bHY6yh-d1e515-d1e119 should match up with the next section that starts with "psalmista psalmus"

Reply:
Sure, can we meet on Monday?

Reply:
resolved.